pkg/engine/jsonContext.go

```diff
@@ -195,3 +199,22 @@ func fetchConfigMap(entry kyverno.ContextEntry, lister dynamiclister.Lister) ([]
 
 	return data, nil
 }
+
+// mapRefactorizer recursively iterates through a map and updates its values in the following way
+// whenever it encounters a string value containing "\n",
+// it converts it into a []string by splitting it by "\n"
+func mapRefactorizer(m map[string]interface{}) map[string]interface{} {
```

Comment:
the name `mapRefactorizer` seems a bit generic to me....can we pick a name that better describes what this does? For example, `parseMultiLineString`?  The other format could be called `parseJSONArray` and ideally both are performed in the same loop.

Reply:
Sure, I'll pick up a more semantically appropriate name 👍🏼